frontend/public/components/nav/_perspective-nav.scss

```diff
@@ -1,17 +1,21 @@
-.oc-nav-group.pf-c-nav__section {
-  margin-top: 0;
-  .pf-c-nav__section-title {
-    padding-bottom: 0;
+.pf-c-nav__list {
+  > li {
```

Comment:
While you're fixing the markup semantics, it would also be great if we could move away from descendent selectors like these and do a better job of sticking with [BEM conventions](http://getbem.com/introduction/) since descendent selectors like these are explicitly tied to the markup structure, and these changes are making the existing rules even *more* explicit.

It would be better to add classes to the individual elements and target those new classes with overrides.  The existing `oc-nav-group` class and rule is one such example (although ideally it would *not* be paired with the PatternFly class because that makes the rule is dependent on the PatternFly class being present).

@sg00dwin or I can assist if needed.

Reply:
I tried to be carefully here because of the two levels of navigation items here and tried a minimal-save-change.

The actual tree is something like `ul > li > section > ul > li`

I can take a second look into the dom structure and try to apply your mentioned BEM conventions.

Reply:
@rhamilto Replaced the `>li` with PatternFly components. Cleaned up the code a little bit. If you find the time, please take a new look and let me know what you think.

Reply:
Nice.  Definitely a step forward instead of a step back.  Good work.